Tests/Clappr_Tests/Classes/Plugin/Core/Drawer/BottomDrawerPluginTests.swift

```diff
@@ -0,0 +1,122 @@
+
+                it("has bottom as position") {
+                    expect(plugin.position).to(equal(.bottom))
+                }
+
+                it("has size with the same width and the half of the height from parentView") {
+                    core.view = UIView(frame: CGRect(x: 0, y: 0, width: 320, height: 100))
+
+                    expect(plugin.size).to(equal(CGSize(width: 320, height: 50)))
+                }
+            }
+
+            describe("#render") {
+                var core: CoreStub!
+                var plugin: BottomDrawerPlugin!
+
+                beforeEach {
+                    core = CoreStub()
+                    plugin = BottomDrawerPlugin(context: core)
+                    core.view = UIView(frame: CGRect(x: 0, y: 0, width: 320, height: 100))
+                }
+
+                it("has origin x on zero and y equal to parentView height") {
+                    plugin.render()
+
+                    expect(plugin.view.frame.origin).to(equal(CGPoint(x: .zero, y: 100)))
+                }
+
+                it("has a view frame size equals to his size") {
+                    plugin.render()
+
+                    expect(plugin.view.frame.size).to(equal(CGSize(width: 320, height: 50)))
+                }
+
+                context("when plugin request a height greater then the limit") {
+                    it("uses the height limit insted") {
+                        core.view = UIView(frame: CGRect(x: 0, y: 0, width: 320, height: 100))
+                        let pluginMock = BottomDrawerPluginMock(context: core)
+
+                        pluginMock.render()
+
+                        expect(pluginMock.view.frame.size).to(equal(CGSize(width: 320, height: 50)))
+                    }
+                }
+            }
+
+            describe("#events") {
+                var core: CoreStub!
+                var plugin: BottomDrawerPlugin!
+
+                beforeEach {
+                    core = CoreStub()
+                    plugin = BottomDrawerPlugin(context: core)
+                    core.view = UIView(frame: CGRect(x: 0, y: 0, width: 320, height: 100))
+                    plugin.render()
+                }
+
+                context("when the showDrawerPlugin event is triggered") {
+                    it("push up the drawer to half of core height") {
+                        core.trigger(.showDrawerPlugin)
+
+                        expect(plugin.view.frame.origin).to(equal(CGPoint(x: .zero, y: 50)))
+                    }
+                }
+
+                context("when the hideDrawer event is triggered") {
+                    it("push down the drawer to core height") {
+                        core.trigger(.hideDrawerPlugin)
+
+                        expect(plugin.view.frame.origin).to(equal(CGPoint(x: .zero, y: 100)))
+                    }
+                }
+
+                context("when the didTappedCore event is triggered") {
+                    it("calls hideDrawer event") {
```

Comment:
I think this should be tested in the `DrawerPluginTests`, since it's an action performed by the `DrawerPlugin` class.
What do you think?